css/style.css

```diff
@@ -282,11 +289,45 @@ footer ul li {
   list-style: none;
   margin: 0 26px 75px;
 }
-footer p {
-  font-size: 18px;
+.copyright {
+  font-size: 0.692rem;
   color: #fff;
   font-weight: 600;
 }
 
+/******************************************
+/* SCREEN BREAKS
+/*******************************************/
+
+@media screen and (max-width: 1000px) {
+  #benefits { width: 100%; padding: 60px 40px; }
```

Comment:
consider resizing the `body` `font-size` within your media queries to control the global font size for the entire page when the screen size changes